package.json

```diff
@@ -1,12 +1,15 @@
 {
-  "name": "node-wpilib-ws",
+  "name": "@wpilib/node-wpilib-ws",
```

Comment:
Do we like the name? This is probably the only opportunity we will get to rename it haha

Reply:
True! I'm fine with the name... especially since I don't have any other good ideas lol